### ISSUE TITLE
Icon text baseline calculation does not match dynamic fonts

### ISSUE THREAD

**Issue by PavelHolec** [kiwicom/orbit-swiftui#108]
The calculation of Icon text baseline does not work with dynamic font sizes.
